src/handlers/shared/start.ts

```diff
@@ -13,7 +12,8 @@ export async function start(context: Context, issue: Context["payload"]["issue"]
   const { taskStaleTimeoutDuration } = config.timers;
   const assignee = context.payload.issue.user.login;
   const userRole = (await getUserRole(context, assignee)) as UserRole;
-  const maxConcurrentTasks = maxTask[userRole];
+  const { maxConcurrentTasks } = config.miscellaneous;
+  const maxTask = maxConcurrentTasks.find(({ role }) => role.toLowerCase() === userRole)?.limit as number;
```

Comment:
How do we handle the case if no role is found? Can we provide some feedback to the user that they haven't configured this role? Do we then default to the lowest default role?

Reply:
For now the default lowest role is contributor

Reply:
Yeah but keep in mind those only take affect if the user inputs no role definitions. If they define three custom roles and the role returned is not one of the three this should be handled via logging feedback and then finding the lowest limit in the config and applying that.